tests/main/interfaces-classic-content-slot/task.yaml

```diff
@@ -0,0 +1,65 @@
+summary: Ensure that the content slots for classic snaps works.
+
+details: |
+    The content-sharing interface interface allows a snap to access contents from
+    other snap. It is possible to have content slots in classic snaps, where
+    confined snaps can connect to.
+
+    A snap which defines the content sharing plug must be shown in the interfaces list.
+    The plug must be autoconnected on install and, as usual, must be able to be
+    reconnected.
+
+# We use classic snaps for the test, so it cannot run on core
+systems: [-ubuntu-core-*]
+
+environment:
+    PROVIDER_SNAP: test-snapd-classic-content-slot
+    CONSUMER_SNAP: test-snapd-content-plug-to-classic-slot
+
+restore: |
+    snap remove --purge test-snapd-content-plug-to-classic-slot
```

Comment:
should be: 
snap remove --purge "$CONSUMER_SNAP"

Reply:
Also snaps are automatically removed during restore, so you don't need to do snap remove.